### PR TITLE
Add set up guide in README

### LINKED ISSUE #18: Create guide on how to set up environment
It's kind of confusing on how to set up the blender and python environment, we should make a guide in the README

### PR DESCRIPTION
**Pull request by JessicaJHee**
Fixes #18

### CODE REVIEW

**Comment by o-farag** [comment]
@JessicaJHee Good work, I really like the use of the venv.
One question, this is what happens when I run `pip install -r requirements`
image
1. Did you mean for the file to have the .txt extension?
2. Are these errors ok i.e. expected (I can still run the app afterwards)
3. If not, were there any other steps you went through first?

**Review by o-farag — CHANGES_REQUESTED** [review]
Could you also add how to deactivate the venv? It's a simple `$ deactivate`  shell command once in the venv.

**Comment by JessicaJHee** [comment]
@o-farag Yep nice catch, it was supposed to have the `.txt` extension.

What version of Python are you using? it should be 3.10.

**Comment by o-farag** [comment]
> @o-farag Yep nice catch, it was supposed to have the `.txt` extension.
> 
> What version of Python are you using? it should be 3.10.

Oh ok I'm using 3.11, maybe the version should be mentioned as well.

File: README.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # LensCrafters-free
 
+#### Project Requirements
+* Node 16
+* Python 3.10
```
Comment (JessicaJHee) [inline README.md:5]: @o-farag mentioned here:

Comment (o-farag) [reply README.md:5]: Woops hehe

**Review by o-farag — APPROVED** [review]
Thank you :)

**Pull request merged**